tests/test_connector.py

```diff
@@ -452,10 +453,40 @@ def test_cleanup3(self):
     def test_tcp_connector_ctor(self):
         conn = aiohttp.TCPConnector(loop=self.loop)
         self.assertTrue(conn.verify_ssl)
+        self.assertIs(conn.expect_fingerprint, None)
         self.assertFalse(conn.resolve)
         self.assertEqual(conn.family, socket.AF_INET)
         self.assertEqual(conn.resolved_hosts, {})
 
+    def test_tcp_connector_ctor_expect_fingerprint_valid(self):
+        valid = '7393fd3aed081d6fa9ae71391ae3c57f89e76cf9'
+        conn = aiohttp.TCPConnector(loop=self.loop, expect_fingerprint=valid)
+        self.assertEqual(conn.expect_fingerprint, valid)
+
+    def test_tcp_connector_expect_fingerprint_invalid(self):
+        invalid = 'a1b2c3'
+        with self.assertRaises(ValueError):
+            aiohttp.TCPConnector(loop=self.loop, expect_fingerprint=invalid)
+
+    def test_tcp_connector_expect_fingerprint(self):
```

Comment:
> Maybe we need to check all three fingerprints formats: md5, sha1, sha256?

[Done](https://github.com/requiredfield/aiohttp/commit/a24960cf89ca156344210617ff374ec890c256bc#diff-323ee80a9a6123b5ea1613be937b1635R474) in a24960c